extensions/phonetic/src/org/openrefine/phonetic/keyers/DaitchMokotoffKeyer.java

```diff
@@ -0,0 +1,15 @@
+package org.openrefine.phonetic.keyers;
+
+import com.google.refine.clustering.binning.Keyer;
+import org.apache.commons.codec.language.DaitchMokotoffSoundex;
+
+public class DaitchMokotoffKeyer extends Keyer {
```

Comment:
Similarly, we should introduce with a comment the concept of Keyer and what it is and why its used.  The tests help somewhat, but useful to really explain here about binning keys.

Reply:
@thadguidry I think that should rather go on the Keyer interface itself